src/main/java/uk/gov/hmcts/reform/divorce/casemaintenanceservice/domain/model/DivorceCaseProperties.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.divorce.casemaintenanceservice.domain.model;
+
+public interface DivorceCaseProperties {
```

Comment:
Please use the following patterm

```suggestion
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public class DivorceCaseProperties {
    public static final String D8_CASE_REFERENCE = "D8caseReference";
```